docs/agents-and-ingestion-tools/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Agents and Ingestion Tools
+nav_order: 100
+has_children: false
+has_toc: false
+---
+
+# Agents and Ingestion Tools
+
+Historically there have been multiple popular agents and ingestion tools that work with Elasticsearch. Some of which are Beats, Logstash, Fluentd, FluentBit, and Open Telemetry. OpenSearch aims to continue to support a broad set of agents and ingestion tools, but not all have been tested or have explicitly added OpenSearch compatibility. To make getting started with OpenSearch easier a [set version YML configuration](https://github.com/opensearch-project/OpenSearch/issues/693) is being added so that you can set OpenSearch 1.x clusters to report open source Elasticsearch 7.10.2. This will enable it to connect with tools that do version checking while need OpenSource compatibilities are added to your tools.
```

Comment:
"Historically,"

" To make getting started with OpenSearch easier a set version YML configuration is being added so that you can set OpenSearch 1.x clusters to report open source Elasticsearch 7.10.2." <- check grammar/fluency (doesn't make sense to me)

"This will enable it to connect with tools that do version checking while need OpenSource compatibilities are added to your tools." <- check grammar/fluency